README.md

```diff
@@ -1,74 +1,142 @@
-**UNDER DEVELOPMENT**
-
 # SOLVCON Development Environment
 
-This project provides a reference runtime environment for developing SOLVCON
-and related projects to reduce unnecessary misunderstandings.  It supports the
-following features:
+**UNDER DEVELOPMENT**
 
-1. Support ubuntu and macOS.
-2. Manage dependencies in the source-code level.  It may build all dependencies
-   from source and retain the source code, so that a debugger can read it.
-3. Allow building a selected set of packages from repository / master.  The
-   minimal set includes cpython, numpy, and pybind11.
-4. Allow multiple copies of the runtime environment and build at the same time.
-   At least support two flavors: release and debug.
+This project is a set of (Bourne again) shell scripts to align the environment
+for developing SOLVCON and related projects in (Ubuntu) Linux and macOS.  The
+features include:
 
-The environment consists of scripts for building dependencies and managing
-them.  More information will be added as the project becomes more complete.
+1. Build dependencies from source and retain the source code for debugger.
+2. Unprivileged installation.  No root permission is required.
+3. Switchable environments.
+4. Download source code from a repository or a tarball.
 
-## Pre-requisite
+# Installation
 
-By using the following command to setup all necessary environment variables:
+Clone the repository to where you want to install, usually somewhere under your
+home directory (e.g., `~/devenv`):
 
+```bash
+git clone git@github.com:solvcon/devenv.git ~/devenv
 ```
-source scripts/init
+
+(Or use https:)
+
+```bash
+git clone https://github.com/solvcon/devenv ~/devenv
 ```
 
-## Advanced Configuration
+## Enabling
```

Comment:
Enabling is a separate step after installation.

Reply:
I removed `DEVENVFLAVORROOT`.  It's too advanced.  I think we should reveal to users who hack into the scripts and find it themselves.